cirro/cli/interactive/download_args.py

```diff
@@ -72,28 +81,41 @@ def ask_dataset_files_list(files: List[File]) -> List[File]:
         ]
     })
 
-    return [
+    selected_files = [
         file
         for file in files
         if file.relative_path in set(answers['files'])
     ]
 
+    if len(selected_files) == 0:
+        if ask(
+            "confirm",
+            "No files were selected - try again?"
+        ):
+            return ask_dataset_files_list(files)
+        else:
+            raise RuntimeWarning("No files selected")
+    else:
+        return selected_files
+
 
 def ask_dataset_files_glob(files: List[File]) -> List[File]:
 
     selected_files = ask_dataset_files_glob_single(files)
```

Comment:
this loop could be a bit simplified as well:

```
while True:
    selected_files = ask_dataset_files_glob_single(files)
    confirmed = ask("confirm", f'Number of files selected: {len(selected_files)} / {len(files):,}')
    if confirmed:
        break
```

Reply:
Similar approach, but avoiding `while True` (d58a4d3)